common/src/chain/transaction/transaction_v1.rs

```diff
@@ -7,7 +7,7 @@ use parity_scale_codec_derive::{Decode as DecodeDer, Encode as EncodeDer};
 
 use super::Transaction;
 
-#[derive(Debug, Clone, PartialEq, Eq, EncodeDer, DecodeDer)]
+#[derive(Debug, Hash, Clone, PartialEq, Eq, EncodeDer, DecodeDer)]
```

Comment:
As discussed in the duplicate thing, please remove the Hash thing from all transaction instances and use the Ideable implementation for that until we find a final solution to this problem.

Reply:
Done